Sources/Fuzzilli/Execution/REPRL.swift

```diff
@@ -20,71 +20,89 @@ import libreprl
 public class REPRL: ComponentBase, ScriptRunner {
     /// Kill and restart the child process after this many script executions
     private let maxExecsBeforeRespawn = 1000
-    
+
     /// Commandline arguments for the executable
     private let processArguments: [String]
-    
+
     /// Environment variables for the child process
     private var env = [String]()
-    
+
     /// Number of script executions since start of child process
     private var execsSinceReset = 0
-    
+
     /// Number of execution failures since the last successfully executed program
     private var recentlyFailedExecutions = 0
-    
+
     private var reprlContext: OpaquePointer? = nil
-    
+
+    /// Buffer to hold scripts, this lets us debug issues that arise if
+    /// previous scripts corrupted any state which is discovered in
+    /// future executions.
+    private var scriptBuffer: [String]? = nil
```

Comment:
I don't think this needs to be an optional. Then those unsafe unwrapping operations (`!`) aren't needed. Maybe depending on the the above comment (`DiagnosticsEvent`), this can also just be a single string which is concatenated to I guess